packages/apps/desktop-viewer-test/package.json

```diff
@@ -0,0 +1,93 @@
+{
+  "name": "@itwin/desktop-viewer-test",
+  "version": "0.1.0",
+  "private": true,
+  "os": [
+    "win32",
+    "linux",
+    "darwin"
+  ],
+  "engines": {
+    "node": ">=10.17.0 <15.0"
+  },
+  "scripts": {
+    "build": "",
+    "build:backend": "tsc -p tsconfig.backend.json",
+    "build:frontend": "cross-env DISABLE_ESLINT=true SKIP_PREFLIGHT_CHECK=true GENERATE_SOURCEMAP=false USE_FAST_SASS=true TRANSPILE_DEPS=false DISABLE_TERSER=true react-scripts build",
+    "clean": "rimraf lib build dist",
+    "copy": "npm run copy:assets && npm run copy:enUS",
+    "copy:assets": "cpx \"./assets/*\" ./build/assets/",
+    "copy:enUS": "cpx \"./build/locales/en/*\" ./build/locales/en-US/",
+    "electron": "electron lib/backend/main.js",
+    "electron:debug": "cross-env NODE_ENV=development electron lib/backend/main.js",
+    "lint": "eslint --config package.json --no-eslintrc \"./src/**/*.{ts,tsx}\" 1>&2",
+    "start": "npm run build:backend && run-p \"start:frontend\" \"electron:debug\"",
+    "start:frontend": "cross-env USE_FULL_SOURCEMAPS=true USE_FAST_SASS=true BROWSER=none SKIP_PREFLIGHT_CHECK=true react-scripts start",
+    "build:pack": "electron-builder --dir",

```

Comment:
I'm hesitant to deliver this with electron-builder as our packager since we do have plans to switch it to something new.

Reply:
this is the test app btw, so it should be ok as long as its not in the cra template

Reply:
whatever is in here would be in the first version of the CRA template....this is just a starting point though. The plan is for v2 to be the more full featured desktop app (with local briefcase support). We can even make this a prerelease version if we think that makes sense.

@calebmshafer  how soon will we have a replacement? I'm hesitant to hold this up if we don't have a solution in place soon

Reply:
With it being a test app, this is fine with me but as part of the CRA template I'm more unsure of if we want to continue down that path.  Let me check in on the status of the installer work for electron forge and see if we can get someone working on it.